spec/models/art_item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe ArtItem, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Thanks for the input. Actually, it was @rbritom who made the changes under the spec folder. He just decided to include those changes in this branch for convenience since we were pair programming. So he's the one to respond to any comment on that ;)